src/stores/RootStore.js

```diff
@@ -25,13 +25,15 @@ import type {
 
 import AppSettingsStore from './AppSettingsStore';
 import AuthStore from './AuthStore';
+import AvatarStore from './AvatarStore';
 import RoutesSettingsStore from './RoutesSettingsStore';
 import DAOEntityStore from './DAOEntityStore';
 import DAOApi from 'brewskey.js-api';
 
 class RootStore {
```

Comment:
I'm still not confident that we need this store... and that we need to use injection instead of just using `import` and singleton.  It seems like a lot of extra work to do it this way.

Reply:
I'm not confident, too. I'll decide after I rewrite intialization flow with organizations, if I'll be able to do it clean with singeltons, I'll fullly remove root store and all injections